src/components/userProfile/components/UserAvatar.js

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 import { Avatar, Card } from 'antd';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
-import { BASIC_INFOMATION, USER_PAGE } from '../constants';
+import { getUserData } from '../selectors';
+
+import { USER_PAGE } from '../constants';
 
 const { Meta } = Card;
 const {
     CLASSNAMES: { AVATAR, AVATAR_CARD, BASIC_INFO, HEADER_TITLE },
     PROFILE_PIX,
     STRINGS: { SQUARE },
     TEXTS: { BASIC_INFO_TEXT },
+    STRINGS: { FIRST_NAME, LAST_NAME, USER_NAME },
 } = USER_PAGE;
 
-export default class UserAvatar extends React.Component {
+class UserAvatar extends React.Component {
+
     render() {
+
+        const { firstName, lastName, username } = this.props.userData;
+
+        const BASIC_INFOMATION = [
+            { description: firstName, title: FIRST_NAME },
+            { description: lastName, title: LAST_NAME },
+            { description: username, title: USER_NAME },
+        ];
```

Comment:
i do not agree with this pattern being here but its okay for now.

Reply:
ok